src/software/jetson_nano/services/motor.cpp

```diff
@@ -177,11 +164,17 @@ void MotorService::setUpMotors()
     checkDriverFault(DRIBBLER_MOTOR_CHIP_SELECT);
     startController(DRIBBLER_MOTOR_CHIP_SELECT, true);
     tmc4671_setTargetVelocity(DRIBBLER_MOTOR_CHIP_SELECT, 0);
-}
 
+    checkEncoderConnections();
+
+    is_initialized = true;
+}
 
-bool MotorService::checkDriverFault(uint8_t motor)
+MotorService::MotorFaultIndicator MotorService::checkDriverFault(uint8_t motor)
 {
+    bool drive_enabled = true;
```

Comment:
If drive is not enabled for some motor, do we want to reset it? Or is the plan to manually reset it by restarting thunderloop?